src/main/java/module-info.java

```diff
@@ -1,6 +1,7 @@
 module org.cryptomator.jsmb {
 	requires org.slf4j;
 	requires static org.jetbrains.annotations;
+	requires org.bouncycastle.provider;
```

Comment:
ok for now, but we should try to add a cmac impl without requiring further dependencies

Reply:
Resolved by b02480fed82b4bcf201c24c296394d505c7c291f.